src/shared/codelens/pythonCodeLensProvider.ts

```diff
@@ -59,75 +70,210 @@ const getLambdaHandlerCandidates = async ({ uri }: { uri: vscode.Uri }): Promise
         })
 }
 
-export function initialize({
+// Add create debugging manifest/requirements.txt containing ptvsd
+const makePythonDebugManifest = async (params: {
+    samProjectCodeRoot: string,
+    outputDir: string
+}): Promise<string | undefined> => {
+    let manifestText = ''
+    const manfestPath = path.join(params.samProjectCodeRoot, 'requirements.txt')
+    if (fileExists(manfestPath)) {
+        manifestText = await readFileAsString(manfestPath)
+    }
+    getLogger().debug(`pythonCodeLensProvider.makePythonDebugManifest params: ${JSON.stringify(params, undefined, 2)}`)
+    if (manifestText.indexOf('ptvsd') < 0 ) { // TODO: Make this logic more robust
+        manifestText += '\nptvsd'
+        const debugManifestPath = path.join(params.outputDir, 'debug-requirements.txt')
+        await writeFile(debugManifestPath, manifestText)
```

Comment:
What happens to this file after debugging? Do we clean it up?

Reply:
It gets written to a temp dir created by `makeBuildDir ` which _should_ get cleaned up by `ExtensionDisposableFiles`. See [here](https://github.com/aws/aws-toolkit-vscode/blob/2eb409568f50aafaff477a62ba9f6ca6d50a29ed/src/shared/codelens/localLambdaRunner.ts#L128).